Neos.ContentRepository.Core/Classes/SharedModel/Node/NodeAggregateIds.php

```diff
@@ -92,6 +90,11 @@ public function merge(self $other): self
         ));
     }
 
+    public function contain(NodeAggregateId $nodeAggregateId): bool
```

Comment:
Shouldn't it be called `contains`?
```suggestion
    public function contains(NodeAggregateId $nodeAggregateId): bool
```

Reply:
I was wondering too, but `NodeAggregateIds` is a plural, so "contain" is gramatically correct I think

Reply:
yey singular makes sense in that case ^^. `$nodeAggregateIds->contain($id)`

Alternative would be `$nodeAggregateIds->has($id)`

Reply:
True